### PR TITLE
Migrate to Font Awesome v5

### LINKED ISSUE #1375: Migrate to font-awesome 5.0
**Is your feature request related to a problem? Please describe.**
Want to upgrade to get more icon fonts

**Describe the solution you'd like**
* Modularize font awesome so we only import what we are using.
`import { faArrowAltCircleRight } from '@fortawesome/free-regular-svg-icons';`

### PR DESCRIPTION
**Pull request by Benjaki2**
## Description
Migrate to Font Awesome v5
Fixes #1375  .

- [x] Update `package.json`
- [x] Update `classNames` and `fontSizes` where necessary
- [x] Replace images with new FA icons where possible

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- [x] I have added necessary documentation (if applicable)
- [x] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] Any dependent changes have been merged and published in downstream modules (if applicable)

## Further comments

Many fonts have visibly changed. So, a lot of them won't match perfectly what is in Prod.

@nasa-gibs/worldview

### CODE REVIEW

**Comment by Benjaki2** [comment]
Thanks for locating those missing icons @edplato Those issues should be fixed by the latest commits.

File: web/css/link.css [hunk 1]
```diff
@@ -135,35 +135,34 @@
...
-  margin-left: 1px;
 }
 
-#social-share .fa-reddit::before {
+#social-share .reddit::before {
   border-radius: 50%;
   position: absolute;
   left: -1px;
-  top: 0;
+  top: -1px;
```
Comment (edplato) [inline web/css/link.css]: This seems to help with the Reddit icon in Chrome. It's strange that's the only social icon out of the four affected.
```suggestion
  top: -2px;
```

**Comment by Benjaki2** [comment]
@edplato It looks link I was able to remove all the the reddit-specific customizations by using a reddit icon without background. It could be a new one. Could you give this one more glance 😁? That suggested change feature is pretty cool.

**Comment by ZachTRice** [comment]
Ah, I remember this reddit icon... There was only an inverse icon available in FA4 (pre-v4.5) so I had to be clever in implementing a solid background for it. Good call simply switching it out @Benjaki2

**Comment by edplato** [comment]
Nice, that icon change did the trick!

**Pull request merged**